README.md

```diff
@@ -1 +1,208 @@
-# Hello-Microverse-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <h3><b>Set up a "Hello Microverse" Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Set up a "Hello Microverse" project <a name="about-project"></a>
+
+
+**Set up a "Hello Microverse" project** is my first project on Microverse Program, It is focused on learning some important issues that will be used along all program experience.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li>HTML</li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li>CSS</li>
+  </ul>
+</details>
+
+<details>
+<summary>Linters</summary>
+  <ul>
+    <li>Linters</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Creating branches on repositories.**
+- **Use of linter on HTML&CSS project.**
+- **Write a correct descriptive README file.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Understand the concept of version control.
+- Clone or make a copy of this repo on your local machine.
+- Follow GitHub flow.
+- A carefully reading of this README.md is required.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+ 
+ ```bash
+ cd my-folder
+ git clone git@github.com:oovillagran/Hello-Microverse-project.git
+```
+
+### Install
+
+Install this project with:
+
+ ```bash
+ npm install
+```
+
+### Usage
+
+To run the project, you can use your favorite browser.
+
+
+### Run tests
+
+To run tests, execute the following command:
+
+```bash
+ npx hint .
+```
+
+### Deployment
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@oovillagran](https://github.com/goovillagran)
```

Comment:
- [ ] Good job fixing the changes requested by the previous reviewer, please note that your GitHub link is not working. the handle is incorrect.  `goovillagran`  take out the `g` as your username is `oovillagran`

Reply:
Hello @wayungi,

Thank you. I already made the required correction.